tox.ini

```diff
@@ -11,46 +11,46 @@ envlist =
     vendorverify
 
 [testenv]
-extras = dev
+deps = -rrequirements-dev.txt
 commands =
     pytest {posargs:-v}
     python setup.py build
 
 [testenv:py37-tinycss2]
-extras = dev,css
+deps = -rrequirements-dev.txt
+extras = css
 commands =
     pytest {posargs:-v}
-    python setup.py build
```

Comment:
I don't think these tox environments need to run `python setup.py build` since it's already run in the regular non-tinycss2 environment and adding tinycss2 to the mix doesn't affect builds at all. So in the interest of reducing test time, I nixed this.